tools/tools.py

```diff
@@ -167,9 +167,16 @@ def check(msg):
         
         
         await jobChannel.send(embed=randomize_colour(embed))
-        
-        
-        
-        
-        
 
+    @commands.command()
+    async def members(self, ctx, rolename):
+        role = discord.utils.get(ctx.message.guild.roles, name=rolename)
+        if role is None:
+            await ctx.send("No such role in the server.")
+            return
+        result = role.members
+        if not result:
+            await ctx.send("No members with such role in the server.")
+            return
+        for member in result:
+            await ctx.send(embed=discord.Embed(description=member.mention, colour=discord.Colour.green()))
```

Comment:
This will send a huge amount of embeds. 
Save member mentions in one string and send that string as a description of the embed.

Check this for reference:
[Line 75](https://github.com/milan252525/LAcogs/blob/e709b86e30ca510578bc9142e23d71d35a47acee/tools/tools.py#L75)

Reply:
Done. My favorite copy paste action, yay.